azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc

```diff
@@ -0,0 +1,349 @@
+= Azure Data Lake Storage Connector Examples - Mule 4
+
+The following Azure Data Lake Storage Connector examples show you how to:
+
+* <<create-file-system-example,Create an Azure file system>>. 
+* <<create-path-example,Create a path inside an Azure file system>>.
+* <<update-path-example,Update a path inside the Azure file system>>. 
```

Comment:
I suggest it should be Azure Data Lake file system. There is another product called Azure Files and it could confuse some people.

Update a path inside **an** Azure file system, there could be many file systems